README.md

```diff
@@ -1,38 +1,69 @@
-# Blog flask extension
+# Ubuntu blog extension
 
-This extension allows you to add a simple blog frontend to your flask app. All the articles
-are pulled from the WordPress API that has the plugin WP-JSON.
+This extension allows you to add a simple frontend section to your flask app. All the articles
+are pulled from the (Ubuntu blog)[https://ubuntu.com/blog] WordPress API using JSON.
 
 This extension provides a blueprint with 3 routes:
+
 - "/": that returns the list of articles
 - "/<slug>": the article page
 - "/feed": provides a RSS feed for the page.
+	
+## How to install
+
+To install this extension as a requirement in your project, you can use PIP;
+
+```bash
+pip install canonicalwebteam.blog
+```
+
+See also the documentation for (pip install)[https://pip.pypa.io/en/stable/reference/pip_install/].
+
 ## How to use
 
+### Templates
+
+The module expects HTML templates at `blog/index.html`, `blog/article.html`, `blog/blog-card.html`, `blog/archives.html`, `blog/upcoming.html` and `blog/author.html`. 
+
+An example of these templates can be found at https://github.com/canonical-websites/jp.ubuntu.com/tree/master/templates/blog.
+
 ### Flask
 
-In your app you can:
+In your app you can then:
 
 ```python
     import canonicalwebteam.blog_extension import BlogExtension
     blog = BlogExtension(app, "Blog title", [1], "tag_name", "/url-prefix")
 ```
+
 If you use the factory pattern you can also:
+
 ```python
     import canonicalwebteam.blog_extension import BlogExtension
     blog = BlogExtension()
     blog.init_app(app, "Blog title", [1], "tag_name", "/url-prefix")
 ```
 
+`BlogExtension()` accepts five parameters;
+
+- (obj) `app`
+- (str) "The title of your blog"
+- (arr) Array of tags from blog entries on which to filter e.g `['1234', '4321']` 
+- (str) Language string e.g. `"lang:en"`
+- (str) "/path/to/display/your/blog"
+
 ### Django
 
 - Add the blog module as a dependency to your Django project
-- Load it at the desired path (f.e. "/blog") in the `urls.py` file
+- Load it at the desired path (e.g. "/blog") in the `urls.py` file
+
 ```python
 from django.urls import path, include
-urlpatterns = [path(r"blog/", include("canonicalwebteam.blog.django.urls"))]
```

Comment:
This was a typo, right?

Reply:
The `r`? `r` means ["raw string"](https://www.journaldev.com/23598/python-raw-string), which is a pattern often used for URL routes, which often contain regular expressions - so it avoids needing to escape regex control characters. I don't have a strong opinion on whether to use `r` here or not.

Reply:
I'll revert it.